files/covjson-reader/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "covjson-reader",
+  "repo": "Reading-eScience-Centre/covjson-reader",
+  "files": {
+    "include": ["*.js"]
```

Comment:
The ideal is to host only the minified file and nothing else. But its up to you if you want to add map files or anything else. I try to push for the min only in most cases